using-valtimo/document/custom-case-headers.md

```diff
@@ -0,0 +1,86 @@
+# Custom case headers
+
+Instead of de default case header which only shows a case name, each case header can be customized, using a
+configuration in the environment file of a front-end implementation.
+
+The custom header consists of blocks that use the Bootstrap column classes to define the block sizes. In this way, any
+layout can be created.
+
+| <!-- -->                                          |
+|---------------------------------------------------|
+| ![Custom case header](img/custom-case-header.png) |
+| <center>A custom case header</center>             |
+
+## Defining a custom case header
+
+For the above screenshot, the configuration in the environment looks like this:
+
+#### **`environment.ts`**
+  ```typescript
+...
+
+export const environment: ValtimoConfig = {
+    ...
+    customDossierHeader: {
+        aansprakelijkstelling: [
+            {
+                labelTranslationKey: 'aansprakelijkstelling',
+                columnSize: 12,
+                textSize: 'xl',
+                customClass: 'font-weight-bold'
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.naam',
+                propertyPaths: ['aanvrager.persoonsgegevens.voorletters', 'aanvrager.persoonsgegevens.achternaam'],
+                columnSize: 6,
+            },
+            {
+                propertyPaths: ['aanvrager.adres.woonplaats'],
+                columnSize: 3,
+                textSize: 'md',
+                customClass: 'text-danger'
+            },
+            {
+                columnSize: 3
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.adres',
+                propertyPaths: ['aanvrager.adres.adres', 'aanvrager.adres.huisnummer', 'aanvrager.adres.postcode', 'aanvrager.adres.woonplaats'],
+                columnSize: 6,
+                textSize: 'sm',
+            },
+            {
+                labelTranslationKey: 'aansprakelijkstelling.telefoonnummer',
+                propertyPaths: ['aanvrager.persoonsgegevens.telefoonnummer'],
+                columnSize: 3,
+                textSize: 'sm',
+                noValueText: 'Nog niet bekend'
+            },
+        ]
+    }
+    ...
+};
+  ```
+
+`customDossierHeader` is the main object that holds all custom case header configurations. Inside of this object, for
+each case for which a custom case header is to be defined, an array can be added under a key with the case id. In the
+example above, a custom case header configuration for a case with the id `aansprakelijkstelling` is added.
+
+Each `customDossierHeaderItem` in `customDossierHeader` can have the following properties:
```

Comment:
In this next section a lot of 'you' is used. I think this is allowed here in the style guide, but you could also use something like this:

```
- `labelTranslationKey`*: a key for the label translation, this refers to a translation key in the translation files. 
This can also be used to just show a text in the header without data from the document by leaving the `propertyPaths` property empty.
```